src/schema/v2/partners.ts

```diff
@@ -119,13 +128,15 @@ const Partners: GraphQLFieldConfig<void, ResolverContext> = {
       partner_categories: partnerCategories,
       ..._options,
     }
-    const cleanedOptions = clone(options)
-    // make ids singular to match gravity :id
-    if (options.ids) {
-      cleanedOptions.id = options.ids
-      delete cleanedOptions.ids
-    }
-    return partnersLoader(cleanedOptions)
+    delete options.ids
+    return partnersLoader(options).then((body) => {
+      const totalCount = body.length
+      return {
+        totalCount,
+        pageCursors: createPageCursors({ page, size }, totalCount),
+        ...connectionFromArray(body, options),
```

Comment:
Technically this should use `connectionFromArraySlice` unless the `ids` param is being passed in.

You have the total count of the records, and `body` is either the _full_ list (which it is in the case of the `ids` param, and `connectionFromArray` is appropriate), or a particular page/slice of the list (when `ids` isn't present).

The main difference noticed will be with regards to accuracy of the pagination related helpers, like `hasNextPage`. 

Wondering what's the use-case? If there's a specific purpose/use in Positron that this root field is being added, maybe we can add that specifically in the resolver (ie- remove any unneeded `ids`, `eligibleForCarousel`, args etc. args). That might generally simplify it. Those could be re-added if/when they are needed.